### ISSUE TITLE
Segfault raptor_solutions.cpp

### ISSUE THREAD

**Issue by koch-t** [hove-io/navitia#377]
On the latest dev snapshot, some occurrence results in a segfault:

```
[Switching to Thread 0x7ffff2fab700 (LWP 13411)]
0x000000000061c51d in navitia::routing::get_walking_solutions (clockwise=false, departs=std::vector of length 94, capacity 94 = {...}, 
    destinations=std::vector of length 117, capacity 117 = {...}, best=..., disruption_active=false, accessibilite_params=..., raptor=...)
    at /opt/navitia/source/routing/raptor_solutions.cpp:229
229                                         if(raptor.data.pt_data->journey_pattern_points[final_rpidx]->stop_point->idx == spid_dep.first) {

print final_rpidx
$1 = 4294967295
```

final_rpidx looks dodgy, will investigate later

**Comment by koch-t** [comment]
Fixed by #379

**Comment by koch-t** [comment]
/v1/journeys?to=stop_point:114774&datetime=20140606T101035&from=stop_point:114265&datetime_represents=departure

Segfaults on dev including #379 

```
./kraken() [0x543cb8]
/lib64/libc.so.6(+0x36160) [0x7fd5efc55160]
./kraken(navitia::routing::getWalkingTime(int, unsigned int, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, bool, bool, navitia::type::AccessibiliteParams const&, navitia::routing::RAPTOR const&)+0x58c) [0x66729c]
./kraken(navitia::routing::get_pareto_front(bool, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, navitia::type::AccessibiliteParams const&, bool, navitia::routing::RAPTOR const&)+0x3fc) [0x66695c]
./kraken(navitia::routing::get_solutions(std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, bool, navitia::type::AccessibiliteParams const&, bool, navitia::routing::RAPTOR const&)+0x5a) [0x66645a]
./kraken(navitia::routing::RAPTOR::compute_all(std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, std::vector<std::pair<unsigned int, navitia::time_duration>, std::allocator<std::pair<unsigned int, navitia::time_duration> > > const&, unsigned int const&, bool, bool, unsigned int const&, unsigned int, navitia::type::AccessibiliteParams const&, std::vector<std::string, std::allocator<std::string> > const&, bool)+0x4a3) [0x66f583]
./kraken(navitia::routing::make_response(navitia::routing::RAPTOR&, navitia::type::EntryPoint const&, navitia::type::EntryPoint const&, std::vector<std::string, std::allocator<std::string> > const&, bool, navitia::type::AccessibiliteParams const&, std::vector<std::string, std::allocator<std::string> >, navitia::georef::StreetNetwork&, bool, bool, unsigned int, unsigned int, bool)+0x57e) [0x65c49e]
./kraken(navitia::Worker::journeys(pbnavitia::JourneysRequest const&, pbnavitia::API)+0xd78) [0x54bf38]
./kraken(navitia::Worker::dispatch(pbnavitia::Request const&)+0x3cb) [0x54d74b]
./kraken(doWork(zmq::context_t&, DataManager<navitia::type::Data>&)+0x228) [0x53b3a8]
/usr/lib64/libboost_thread.so.1.52.0(+0x12727) [0x7fd5f22e4727]
/lib64/libpthread.so.0(+0x7d96) [0x7fd5effd5d96]
/lib64/libc.so.6(clone+0x6d) [0x7fd5efd0fd1d]
```

If i do it on a debug-build (difference besides the -g and 03 flag is that Clang was used on the release build) i get a segfault at a different place:

```
handle_vj<navitia::routing::VisitorWalkingTime> (current_jpp_idx=<optimized out>, accessibilite_params=..., raptor_=...)
    at /usr/lib/gcc/x86_64-pc-linux-gnu/4.7.3/include/g++-v4/bits/stl_iterator.h:728
728           __normal_iterator(const _Iterator& __i) : _M_current(__i) { }
```

which appears to be make #381 a duplicate issue
